staging/src/k8s.io/client-go/tools/remotecommand/remotecommand.go

```diff
@@ -83,16 +92,11 @@ func NewSPDYExecutor(config *restclient.Config, method string, url *url.URL) (Ex
 
 // NewSPDYExecutorForProtocols connects to the provided server and upgrades the connection to
 // multiplexed bidirectional streams using only the provided protocols. Exposed for testing, most
-// callers should use NewSPDYExecutor.
-func NewSPDYExecutorForProtocols(config *restclient.Config, method string, url *url.URL, protocols ...string) (Executor, error) {
-	wrapper, upgradeRoundTripper, err := spdy.RoundTripperFor(config)
-	if err != nil {
-		return nil, err
-	}
-	wrapper = transport.DebugWrappers(wrapper)
```

Comment:
Do you need to move this up into `NewSPDYExecutor`?

Reply:
already done inside spdy.RoundTripperFor -> transport.HTTPWrappersForConfig

Reply:
I see that now, thanks.